src/transformState.js

```diff
@@ -62,7 +62,26 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const objectTransform of transforms) {
+    const action = objectTransform.operation;
+    const aimProperties = objectTransform.properties;
```

Comment:
усложняешь
можно так
```suggestion
  for (const transform of transforms) {
    transform.operation;
    transform.properties;
```
или так
```suggestion
  for (const { properties, operation } of transforms) {
```